### PR TITLE
Daily_Naive_Darts

### PR DESCRIPTION
**Pull request by Mohamadkhalil91**

### CODE REVIEW

File: models/Daily_naive_darts/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+enfobench>=0.3.3
```
Comment (attila-balint-kul) [inline models/Daily_naive_darts/requirements.txt]: Please update the version to 0.3.5, I added the helper function in this package

File: models/Daily_naive_darts/requirements.txt [hunk 2]
```diff
@@ -0,0 +1,2 @@
+enfobench>=0.3.3
+darts>=0.27.0
```
Comment (attila-balint-kul) [inline models/Daily_naive_darts/requirements.txt]: please use the u8darts package as it is much smaller than the darts package (it does not have all the pytorch dependencies included that we do not need for this example)

File: models/Daily_naive_darts/src/main.py [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+                    name="Mohamad Khalil",
+                    email="coo17619@newcastle.ac.uk"
+                )
+            ],
+            type=ForecasterType.point,
+            params={},
+        )
+
+
+    def periods_in_duration(self, ts, duration) -> int:
```
Comment (attila-balint-kul) [inline models/Daily_naive_darts/src/main.py]: You can remove this as I added this as a helper to v0.3.5. Please check the updated examples to import it and use it.

File: models/Daily_naive_darts/src/main.py [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+from enfobench import AuthorInfo, ModelInfo, ForecasterType
+from enfobench.evaluation.server import server_factory
+from enfobench.evaluation.utils import create_forecast_index
+
+class NaiveModel:
+
+    def info(self) -> ModelInfo:
+        
+        return ModelInfo(
+            name="Darts.dailyNaive",
```
Comment (attila-balint-kul) [inline models/Daily_naive_darts/src/main.py]: Please use CamelCase for the name to be consistent with all other models. "Darts.DailyNaive"

File: models/Daily_naive_darts/requirements.txt (hunk 1 above)
Comment (Mohamadkhalil91) [reply models/Daily_naive_darts/requirements.txt]: I have updated the version to 0.3.5. thanks,

File: models/Daily_naive_darts/requirements.txt (hunk 2 above)
Comment (Mohamadkhalil91) [reply models/Daily_naive_darts/requirements.txt]: I have updated the package to u8darts

File: models/Daily_naive_darts/src/main.py (hunk 4 above)
Comment (Mohamadkhalil91) [reply models/Daily_naive_darts/src/main.py]: I have updated the name of the model to "DailyNaiveModel"

File: models/Daily_naive_darts/src/main.py (hunk 3 above)
Comment (Mohamadkhalil91) [reply models/Daily_naive_darts/src/main.py]: I have removed the periods_in_duration function, and updated the script accordingly,

**Comment by attila-balint-kul** [comment]
Great. It is best practice to pin the major dependencies in a docker image so that it is reproducible.
For this just use "u8darts==0.27.0" and we are good to go in adding it to the dashboard.

**Comment by attila-balint-kul** [comment]
also just to keep things consistent, could you change the folder name from 'Daily_naive_darts' to 'dt-daily-naive'

**Comment by Mohamadkhalil91** [comment]
I have updated the folder name to dt-daily-naive

**Review by attila-balint-kul — APPROVED** [review]

**Comment by Mohamadkhalil91** [comment]
Great, thanks.

**Pull request merged**